Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -259,9 +259,8 @@ - (CGSize)_sizeWithAttributes:(NSDictionary<NSString*, id>*)attributes constrain
         size.width = std::numeric_limits<CGFloat>::max();
     }
 
-    if (size.height == 0.0) {
-        size.height = std::numeric_limits<CGFloat>::max();
-    }
+    // Return the height that the text actually requires rather than the constrained size
+    size.height = std::numeric_limits<CGFloat>::max();
```

Comment:
also,please verified with MM. check with the introduction label on first screen.

---
In reply to: [99942712](https://github.com/Microsoft/WinObjC/pull/1917#discussion_r99942712) [](ancestors = 99942712,99671953)